Utils/utils/action_utils_wait.py

```diff
@@ -0,0 +1,30 @@
+from time import sleep
+
+
+from sekoia_automation.action import Action
+from sekoia_automation.exceptions import MissingActionArgumentError
+
+
+class UtilsWait(Action):
+    """
+    Action to wait N seconds
+    """
+
+    MAX_TIME_TO_WAIT = 3600  # 1 hour
+
+    def run(self, arguments) -> None:
+        time_to_wait = arguments.get("duration")
+
+        if time_to_wait is None:
+            raise MissingActionArgumentError("duration")
```

Comment:
I suggest defining a default value. However, as the json-schema required this field, it should never be none.
```suggestion
        time_to_wait = arguments.get("duration", 0)
```